src/api/app/models/project/exceptions.rb

```diff
@@ -0,0 +1,24 @@
+module Project::Exceptions
```

Comment:
Now that this file contains errors, it should be renamed to `errors.rb`.

Could we also have a line between each error class?

I really like this idea. We should do it for other models in other PRs!